engine/dlib/src/dlib/job_thread.cpp

```diff
@@ -89,16 +89,19 @@ static void PutDone(JobThreadContext* ctx, JobItem* item)
 static void JobThread(void* _ctx)
 {
     JobThreadContext* ctx = (JobThreadContext*)_ctx;
-    while (dmAtomicGet32(&ctx->m_Run) != 0)
+    while (true)
     {
-        JobItem item;
+        JobItem item = {};
```

Comment:
Between the loop check and the dmConditionVariable::Wait(), the main thread set `run=0` and also signalled the wakeup condition...